app/controllers/questions/optional_consent_controller.rb

```diff
@@ -1,6 +1,5 @@
 module Questions
   class OptionalConsentController < QuestionsController
```

Comment:
Removed requirememnt for being logged in here because we don't log in AtCapacity people (we dont want them to accidentally be able to continue through the process) but we do have them do the optional consent pages.

Reply:
It was kind of an arbitrary decision to show the at capacity people the second optional consent page and i think you're right -- it doesnt really make sense to have them get this far if we can't serve them.

I THINK the reason it ended up working this way is because we take in birthday and SSN on the legal consent page, and we cant check to see if they're duplicated until we have that information.

I could move the at capacity page before this optional consent, but that also felt odd to me. 

I feel like this whole flow could use some more thought because essentially whats happening here is that these people are consenting to service so they will show up in the hub as not assigned to any vita partner and we're telling the clients we can't help them (show them the DIY link for tax slayer), but I'm worried it will be confusing to see them in the hub. Maybe we shouldnt mark them as consented to service if we cant route them either
but i also kind of hate that we're making people do like 10 things (validate phone number or email, read these consent forms) when we wont be doing anything with their info.